tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
-    "target": "ES2020",
-    "module": "commonjs",
+    "target": "ES2022",
+    "module": "ESNext",
```

Comment:
Another option would be rather than converting this to ESM, just use an inline import for ops-core in the few places it's used.  I believe we would still need to set the node engine version minimum bc import wasn't introduced until node 16

Reply:
kk I think I like the second option, I'll get that change in